File: java/listas/edu/itszapopan/edd/Nodo.java
package edu.itszapopan.edd;

public class Nodo {

   protected String contenido;
   protected Nodo sig;

   public Nodo() {
     contenido = null;
     sig = null;
   }

   public Nodo(String cont, Nodo s) {
     contenido = cont;
     sig = s;
   }

   public Nodo(String cont) {
     contenido = cont;
     sig = null;
   }

   public Nodo getSig() 
   {
     return sig;
   }

   public String getContenido()
   {
     return contenido;
   }

   public void setSig(Nodo sig) {
     this.sig = sig;
   }

   public void setContenido(String c) {
     contenido = c;
   }


}

File: cpp/listas/nodo.cpp
#include "nodo.h"

namespace edu_itszapopan_edd  {


Nodo::Nodo() {
contenido = null;
sig = null;
}

Nodo(String cont, Nodo s) {
contenido = cont;
sig = s;
}

Nodo(String cont) {
contenido = cont;
sig = null;
}

Nodo getSig() 
{
return sig;
}

String Nodo::getContenido()
{
return contenido;
}

void Nodo::setSig(Nodo sig) {
this.sig = sig;
}

void setContenido(String c) {
contenido = c;
}




};
















